CHANGES.rst

```diff
@@ -14,6 +14,15 @@ New features:
 
 Bug fixes:
 
+http://localhost:8080/manage
+
+http://localhost:8080/manage
+
```

Comment:
@thet is this double "http://localhost:8080/manage" something you left by mystake?